app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -8,29 +8,55 @@ def initialize(identity)
   end
 
   def user_info
-    user = identity.user
-
-    {
+    info = {
       sub: identity.uuid,
       iss: root_url,
-      email: user.email,
+      email: identity.user.email,
```

Comment:
I'm not sure I see the point? What goal did you have in mind by delegating? Wouldn't that just add noise to the `Identity` class?